### PR TITLE
cloc: add --vcs example

### PR DESCRIPTION
**Pull request by jduan**
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

This PR adds an example to ignore files that are ignored by the VCS, such as git.

### CODE REVIEW

File: pages/common/cloc.md [hunk 1]
```diff
@@ -14,3 +14,7 @@
 - Compare 2 directory structures and count the differences between them:
 
 `cloc --diff {{/directory/one}} {{/directory/two}}`
+
+- Ignore files that are ignored by VCS, such as git:
```
Comment (zdroid) [inline pages/common/cloc.md]:
```suggestion
- Ignore files that are ignored by VCS, such as .git/ directory:
```

Comment (jduan) [reply pages/common/cloc.md]: Thanks for your response! How about this?

    Ignore files that are ignored by VCS, such as files specified in ".gitignore":

That's closer to what I meant to say.

Comment (jduan) [reply pages/common/cloc.md]: BTW, I signed the CLA.

Comment (zdroid) [reply pages/common/cloc.md]: Remove the quote marks and it'll be fine.

File: pages/common/cloc.md [hunk 2]
```diff
@@ -14,3 +14,7 @@
...
+
+`cloc --vcs git {{/path/to/directory}}`
```
Comment (agnivade) [inline pages/common/cloc.md]: Remove preceding `/`

File: pages/common/cloc.md (hunk 1 above)
Comment (jduan) [reply pages/common/cloc.md]: Done. Please take a look.

File: pages/common/cloc.md (hunk 2 above)
Comment (jduan) [reply pages/common/cloc.md]: The leading / is consistent with the existing examples in the file.

Comment (zdroid) [reply pages/common/cloc.md]: The file is quite old, that's why. It definitely needs to be updated according to the newer standards.

Comment (agnivade) [reply pages/common/cloc.md]: Yes, let's change all of the examples while we are here.

Comment (jduan) [reply pages/common/cloc.md]: Got it. I've removed the leading / from all the paths in this file.

Comment (ivanhercaz) [reply pages/common/cloc.md]: @agnivade, I am going to resolve this review due all the leading `/` have been removed of the paths in this file. Excuse me the interference.

File: pages/common/cloc.md (hunk 1 above)
Comment (ivanhercaz) [reply pages/common/cloc.md]: @zdroid, I am going to resolve this review due @jduan change it as it was requested. Excuse me the intereference.

File: pages/common/cloc.md (hunk 2 above)
Comment (ivanhercaz) [reply pages/common/cloc.md]: I tried to approve this to merge this PR, but still seems to appear this issue as a pending change request by @agnivade. Anyone knows why and what could be the solution?

Comment (agnivade) [reply pages/common/cloc.md]: Yes, unless you are an org member and a pending request is there, you cannot force push.

**Comment by jduan** [comment]
@zdroid @ivanhercaz @agnivade Thanks everyone!

File: pages/common/cloc.md (hunk 2 above)
Comment (ivanhercaz) [reply pages/common/cloc.md]: All right! Thank you for this information, @agnivade. Nice to know it for the next times with the same case.

**Pull request merged**